Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,269 @@
+            }
         }
 
         /// <summary>
         /// Return name and loss of a person with a biggest loss (balance change negative).
         /// </summary>
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            CultureInfo culture = CultureInfo.CreateSpecificCulture("");
+            culture.NumberFormat.CurrencyNegativePattern = 1;
+            culture.NumberFormat.CurrencyGroupSeparator = "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0) { return "N/A."; }
+            ParsePeopleAndBalance(peopleAndBalances);
+            float biggestLoss = GetBiggestLoss();
+            if (biggestLoss >= 0) { return "N/A."; }
+            string name = GetNameOfBiggestLoss(biggestLoss);
+            string formattedBalance = string.Format(culture, "{0:C0}", biggestLoss);
+            return $"{name} lost the most money. {formattedBalance}.";
+        }
+
+        // Helper method to return the name of the person who has the biggest loss.
+        private static string GetNameOfBiggestLoss(float biggestLoss)
+        {
+            string result = "";
+            for (int i = 0; i < allBalances.Length; i++)
+            {
+                int index = 0;
+                if(allBalances[i].Length == 0) { continue; }
+                while (index != allBalances[i].Length - 1)
+                {
+                    if (allBalances[i][index + 1] - allBalances[i][index] == biggestLoss)
+                    {
+                        result = allNames[i];
+                        break;
+                    }
+                    index++;
+                }
+            }
+            return result;
+        }
+
+        // Helper method for finding the biggest loss ever.
+        private static float GetBiggestLoss()
+        {
+            float biggestLoss = 0;
+            for (int i = 0; i < allBalances.Length; i++)
+            {
+                float tempLoss = 0;
+                int index = 0;
+                if(allBalances[i].Length == 0) { continue; }
+                while (index != allBalances[i].Length - 1)
+                {
+                    tempLoss = allBalances[i][index + 1] - allBalances[i][index];
+                    if (tempLoss < biggestLoss) { biggestLoss = tempLoss; }
+                    index++;
+                }
+            }
+            return biggestLoss;
         }
 
         /// <summary>
         /// Return name and current money of the richest person.
         /// </summary>
         public static string FindRichestPerson(string[] peopleAndBalances)
         {
-            return "";
+            CultureInfo culture = CultureInfo.CreateSpecificCulture("");
+            culture.NumberFormat.CurrencyNegativePattern = 1;
+            culture.NumberFormat.CurrencyGroupSeparator = "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0) { return "N/A."; }
```

Comment:
this code looks a lot like a blob. Use balnk lines to separate unrelated blocks of code. Also format if statements into a conventional way
```
if(condition
{
  // code
}
```
It really hits to readability if you have it all in one line. The same line both checks a condition and does something if it's true. It should do 1 thing per line.